src/driver/DriverUtils.ts

```diff
@@ -45,8 +45,11 @@ export class DriverUtils {
      *
      * @return An alias allowing to select/transform the target `column`.
      */
-    static buildColumnAlias({ maxAliasLength }: Driver, alias: string, column: string): string {
-        const columnAliasName = alias + "_" + column;
+    static buildColumnAlias({ maxAliasLength, options }: Driver, alias: string, column: string): string {
+        let columnAliasName = alias + "_" + column;
+        if(options.type === "oracle") {
+            columnAliasName = columnAliasName.toUpperCase();
+        }
```

Comment:
If this needs to be the case shouldn't this be done within the oracle query runner?

Reply:
Yeah, ideally within the runner so it's self contained, this is an early draft as I came across issues but definitely not the final solution